examples/landing-zone-v2/tier4/managed-instance-group/README.md

```diff
@@ -0,0 +1,40 @@
+# Managed Instance Group
+
+This example creates a simple managed instance group for an debian instance template with nginx.
+
+## IAM
+The following permissions are required on the projects' allowed subnets until the [config connector resource](https://cloud.google.com/config-connector/docs/reference/resource-docs/compute/computeinstancegroupmanager) allows for specifying the MIG to use a specific service account when creating VMs. Currently, the `spec.serviceAccountRef` throws an error `Invalid value for field ''resource.serviceAccount'': ''tier4-sa@project-id.iam.gserviceaccount.com''. service account support is not yet available, invalid'`
```

Comment:
@alaincormier-ssc do we have an internal bug created to track this ?

Reply:
we do now :)
5802